packages/vite/src/consts.ts

```diff
@@ -0,0 +1,3 @@
+export const compiledVocabFileFilter = /\.vocab[\\/]index\.(?:ts|js|cjs|mjs)$/;
+export const virtualModuleId = 'vocab-virtual-module';
```

Comment:
IMO we should follow [vite's virtual module convention](https://vite.dev/guide/api-plugin.html#virtual-modules-convention) here and for the `resolveId` and `load` plugin hooks.
```suggestion
export const virtualModuleId = 'virtual:vocab';
```